stack-8.8.3.yaml

```diff
@@ -28,7 +28,7 @@ extra-deps:
 - lsp-test-0.11.0.2
 - monad-dijkstra-0.1.1.2
 - opentelemetry-0.4.2
-- ormolu-0.0.5.0
+- ormolu-0.1.2.0
```

Comment:
`lts-16.3` actually has `ormolu-0.1.2.0`. However, I'm guessing you want to pin this here so you can bump the stackage version without bumping ormolu? Or should I remove this line?

Reply:
Yeah that is fine! Thank you!